themes/upbound/assets/scss/_hints.scss

```diff
@@ -78,28 +82,22 @@ $hint-note-background:          rgba(0,176,255,.1);
         }
     }
 
-    &.tip {
-        border-left-color: #{$hint-tip};
+    // &.tip {
+    //     border-left-color: #{$hint-tip};
 
-        >.admonition-title {
-            background-color: #{$hint-tip-background};
-        }
-    }
+    //     >.admonition-title {
+    //         background-color: #{$hint-tip-background};
+    //     }
+    // }
```

Comment:
I like the new call out styles. Only comment is that [tips](https://docs-git-fork-teich-designv3-upboundio.vercel.app/contribute/features/#hints-and-alert-boxes) are now grey, I think we should make them a "positive" color.

Reply:
@shanecmiller23 feedback above ^. I tend to agree but you are the design expert.

Reply:
I don't feel strongly on this. I'd say gray works for now but I will tinker with this after this commit